plugins/auth-node/api-report.md

```diff
@@ -62,6 +68,8 @@ export interface AuthProviderRouteHandlers {
 export interface AuthProvidersExtensionPoint {
   // (undocumented)
   registerProvider(options: AuthProviderRegistrationOptions): void;
+  // (undocumented)
+  setAuthOwnershipResolver(ownershipResolver: AuthOwnershipResolver): void;
```

Comment:
Let's make this a separate extension point as it doesn't really below with auth providers, perhaps `SignInExtensionPoint`, or even more direct `OwnershipResolutionExtensionPoint`?